test/test_sparse_csr.py

```diff
@@ -1865,11 +1866,13 @@ def test_addmm_all_sparse_csr(self, device, dtype, layout):
         m2 = torch.randn(50, 25, device=device).to(dtype)
         _test_addmm_addmv(self, torch.addmm, M, m1, m2, layout=layout, mode="all_sparse")
 
-        # Test beta=0, M=nan
-        M = torch.full((10, 25), float('nan'), device=device).to(dtype)
-        m1 = torch.randn(10, 50, device=device).to(dtype)
-        m2 = torch.randn(50, 25, device=device).to(dtype)
-        _test_addmm_addmv(self, torch.addmm, M, m1, m2, beta=0, layout=layout, mode="all_sparse")
+        # TODO: handle M=nan in Eigen
```

Comment:
What does this one mean?

Reply:
It was failing due to a missed `beta==0`  check in `addmm_out_sparse_csr_native_cpu`. Fixed in https://github.com/pytorch/pytorch/pull/101814/commits/c6a3e334971b055d14e17ec6e9e1d05705f53364.